examples/demo-readme.sh

```
@@ -73,6 +73,26 @@ show 'pipelock integrity check ./workspace'
+pause
+
+# --- 6: Git diff scanning ---
+step "6. Git scan: Catch secrets in diffs"
+printf 'diff --git a/.env b/.env\n--- /dev/null\n+++ b/.env\n@@ -0,0 +1 @@\n+ANTHROPIC_API_KEY=sk-ant-api03-FAKEFAKEFAKEFAKEFAKE\n' | \
+    pipelock git scan-diff --config "$DEMO_TMPDIR/pipelock.yaml" 2>&1 || true
+pause
+
+# --- 7: Config generation ---
+step "7. Generate: Starter config in one command"
+show 'pipelock generate config --preset strict'
+pipelock generate config --preset strict 2>&1 > "$DEMO_TMPDIR/strict.yaml"
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)